scripts/dev/templates/agent/Dockerfile.template

```diff
@@ -1,29 +1,32 @@
+ARG imagebase
+FROM ${imagebase} as base
+
 FROM {{base_image}}
 
 {% block packages -%}
 {% endblock -%}
 
 RUN mkdir -p agent \
-    && curl --fail --retry 3 --silent https://mciuploads.s3.amazonaws.com/mms-automation/mongodb-mms-build-agent/builds/automation-agent/prod/mongodb-mms-automation-agent-{{agent_version}}.{{agent_distro}}.tar.gz -o agent/mongodb-agent.tar.gz \
-    && tar xfz agent/mongodb-agent.tar.gz \
+    && mkdir -p /var/lib/mongodb-mms-automation \
+      && mkdir -p /var/log/mongodb-mms-automation/ \
+      && chmod -R +wr /var/log/mongodb-mms-automation/ \
+      # ensure that the agent user can write the logs in OpenShift
+      && touch /var/log/mongodb-mms-automation/readiness.log \
+      && chmod ugo+rw /var/log/mongodb-mms-automation/readiness.log
+
+
+COPY --from=base /data/mongodb-agent.tar.gz agent
```

Comment:
Can we use absolute directories for the destination? Or it will depend on current WORKDIR